src/integration-test/java/org/polyfill/PolyfillControllerIT.java

```diff
@@ -0,0 +1,79 @@
+package org.polyfill;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.polyfill.configurations.AppConfig;
+import org.polyfill.util.IntegrationTestingUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Created by smo on 4/11/17.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {AppConfig.class})
+@WebAppConfiguration
+public class PolyfillControllerIT {
+
+    private final String UA_CHROME = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3070.0 Safari/537.36";
```

Comment:
I see that we use this kind of string a lot for testing, I was think about use a final class like this
```java
public final class UA  {
  public static final String UA_CHROME = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3070.0 Safari/537.36";
  public static final String UA_FIREFOX = "bar";
}
```
then  we can use it like 

```java
import UA.*;
// ...

if(variable.equals(UA_CHROME)){
//...
}    
```